docs/.vuepress/handsontable-manager/register.js

```diff
@@ -77,11 +77,22 @@ function createRegister() {
 
           if (rootExampleElement) {
             const examplePresetType = rootExampleElement.getAttribute('data-preset-type');
-
-            register.add(createDestroyableResource(examplePresetType, {
+            const tabsComponent = closest(this.rootElement, '.tabs-component');
+            const allTabs = tabsComponent.querySelectorAll('a');
+            const codeTab = Array.from(allTabs).find(tab => tab.innerText === 'Code'); // Usually first tab
```

Comment:
When we decide to change the tab name from `Code` to `The source code` then the code stop working. WDYT to move the logic to the `docs/.vuepress/theme/components/Page.vue` component. I think there you can iterate throught all tabs without touching the DOM.

Reply:
You are right that `Page.vue` have access to components and it would be nice to do destroy instance without reading from the DOM. However, I can see the problem that `docs/.vuepress/handsontable-manager/register.js` file is the first place where code snippet is already executed and I have access to instance of Handsontable (and destroyer created by factory, stored in the `Set`). I don't have problem with accessing tabs in another place, but I can see a problem with catching the instance.

I can't see possibility to pass it to another place. Do you have any idea how to do it? 

Btw. alternatively, to skip basing on the name of DOM element representing the tab I can check `href` (i.e. `#code-tab-example1`) property.

Reply:
I changed little bit a way how DOM element is found: https://github.com/handsontable/handsontable/pull/9827/commits/8e55f5d2788e3a317bbc7f570bc966a457142115